config/webpack.config.js

```diff
@@ -87,7 +88,8 @@ if (__DEV__) {
   debug('Enabling plugins for live development (HMR, NoErrors).')
   webpackConfig.plugins.push(
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin()
+    new webpack.NoErrorsPlugin(),
+    new OpenBrowserPlugin({ url: project.compiler_public_path })
```

Comment:
[project.compiler_public_path](https://github.com/davezuko/react-redux-starter-kit/blob/master/config/environments.config.js#L12) should provide the whole url: `http://${config.server_host}:${config.server_port}/`.